src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -67,6 +70,26 @@
         };
     }
 
+    public static Interview[] getInterviewPositions() {
+        Applicant[] samplePersons = getSamplePersons();
+        Position[] samplePositions = getSamplePositions();
```

Comment:
Why is this called `getInterviewPositions()` shouldn't it be `getSampleInterviews()` like the other methods?

Reply:
Thanks for pointing this out! It's indeed a typo. Will fix it now.